src/vit-servicing-station/doc/api/v0.yaml

```diff
@@ -205,6 +205,36 @@ paths:
         "404":
           description: The requested challenge was not found
 
+  /api/v0/challenges/{id}/{voter_group_id}:
```

Comment:
do we usually add [tests ](https://github.com/input-output-hk/catalyst-core/blob/main/src/vit-servicing-station/vit-servicing-station-tests/src/tests/rest/challenges.rs)for new endpoints? or is it redundant?

Reply:
+1 @cong-or we should definitely satisfy decent automation coverage and since this is new endpoint we should add at least happy path for ti